packages/cli/src/eventbus/MessageEventBusDestination/MessageEventBusDestinationWebhook.ee.ts

```diff
@@ -108,6 +109,7 @@ export class MessageEventBusDestinationWebhook
 		if (foundCredential) {
 			const timezone = config.getEnv('generic.timezone');
 			const credentialsDecrypted = await this.credentialsHelper?.getDecrypted(
+				{} as IWorkflowExecuteAdditionalData,
```

Comment:
Just to be sure, no chance this will break right?

Reply:
Good catch. This would have caused issues. Should be fixed now.